<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thermal Images Processor CIP V1.3</string>
  </property>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TIP CIP V1.3</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_controles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Controls</string>
   </property>
   <widget class="QPushButton" name="pb_addimage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/picture_add.png</normaloff>:/Images/picture_add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_deleteitem">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/picture_delete.png</normaloff>:/Images/picture_delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_addimage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add 
 image(s)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_deleteimage">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete
image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_deleteall">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/pictures.png</normaloff>:/Images/pictures.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_deleteall">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete
 all images</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_generate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/gear_in.png</normaloff>:/Images/gear_in.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_generate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate
 results</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_saveresults">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/folder_picture.png</normaloff>:/Images/folder_picture.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_generate_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select output
 directory</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_excel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add reflected
temperature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_addimage_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/table_excel.png</normaloff>:/Images/table_excel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_clean">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/draw_eraser.png</normaloff>:/Images/draw_eraser.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="l_generate_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean output 
 windows</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_selectedimg">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Input images selected by user</string>
   </property>
   <widget class="QListView" name="lst_imagefile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_visualizer">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Input and output image display</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>271</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_output">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Output images</string>
   </property>
   <widget class="QListView" name="lst_outfiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_outputdir">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Output directory for results</string>
   </property>
   <widget class="QListView" name="lst_outdir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_status">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Processing status</string>
   </property>
   <widget class="QProgressBar" name="pbar_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_about">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>30</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/information.png</normaloff>:/Images/information.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_sponsors">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>721</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Sponsored by</string>
   </property>
   <widget class="QLabel" name="cip_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>701</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Images/logos.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>gb_controles</zorder>
  <zorder>l_title</zorder>
  <zorder>line</zorder>
  <zorder>gb_selectedimg</zorder>
  <zorder>gb_visualizer</zorder>
  <zorder>gb_output</zorder>
  <zorder>gb_outputdir</zorder>
  <zorder>gb_status</zorder>
  <zorder>pb_about</zorder>
  <zorder>gb_sponsors</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
